python/src/ir.cc

```diff
@@ -1906,7 +1906,15 @@ void init_triton_ir(py::module &&m) {
                 setupTritonDiagnosticHandler(context);
             if (failed(self.run(mod.getOperation())))
               throw std::runtime_error("PassManager::run failed");
+
+            if (!repro_suffix.empty() && !reproducerPath.empty() &&
+                reproducerPath != "-" &&
+                llvm::sys::fs::copy_file(reproducerPath,
+                                         reproducerPath + repro_suffix)) {
+              throw std::runtime_error("PassManager::run failed (repro temp)");
```

Comment:
why not change the path, line 1870 instead of doing a copy?

Reply:
I would just drop it, I'm not sure it has a lot of value. People usually know the order of the pass managers

Reply:
Sounds good, also improved the test.